src/os/bluestore/bluestore_types.h

```diff
@@ -156,7 +156,8 @@ struct denc_traits<vector<bluestore_pextent_t>> {
   static void bound_encode(const vector<bluestore_pextent_t>& v, size_t& p) {
     p += sizeof(uint32_t);
     size_t per = 0;
-    denc(*(bluestore_pextent_t*)nullptr, per);
+    bluestore_pextent_t tmp;
```

Comment:
Yeah, the use of nullptr here is deliberate.

I'm not sure how we would static_assert here, or how we would otherwise get the same end result (automatic detection of instance-dependent bounds)...